### PR TITLE
add facerecognition

### PR DESCRIPTION
**Pull request by szaimen**
needed:
- [x] NC21 compatibility https://github.com/matiasdelellis/facerecognition/releases

Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

**Comment by szaimen** [comment]
@enoch85 yes, I know. It is just another additional apps script but I promise this will be the last one for a while.
And this is already much more stable and has much less dependencies as the pdfdraw app, so I think that it fits into the additional apps menu quite nicely. And yes, I think this makes a nice addition to all the other apps, hence I've created it 😉

**Comment by szaimen** [comment]
This script is finally ready and was successfully tested on a NC21 test-VM :)

File: apps/face-recognition.sh [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    # Show successful uninstall if applicable
+    removal_popup "$SCRIPT_NAME"
+fi
+
+# Inform about dependencies
+msg_box "Please note that the $SCRIPT_NAME app needs an additional PHP dependency \
+to work which will need to be installed from an external repository.
+This can set your server under risk."
+if ! yesno_box_yes "Do you want to install the required dependency?
+If you choose 'No', the script will get exited."
```
Comment (enoch85) [inline apps/face-recognition.sh:85]: The installation will be aborted.

Comment (szaimen) [reply apps/face-recognition.sh]: done

**Pull request merged**